token_utils/api.py

```diff
@@ -15,3 +16,15 @@ def sign_token_for(lms_user_id, expires_in_seconds, additional_token_claims):
         additional_token_claims (dict): Additional claims to include in the token.
     """
     return create_jwt(lms_user_id, expires_in_seconds, additional_token_claims)
+
+
+def unpack_token_for(token, lms_user_id):
+    """
+    Unpack and verify a signed JWT token. Validate the user, block, and expiration.
```

Comment:
I think block validation could be handled out in MST-1600 - its going to be a bit fiddly because the block could be be a child of the block in the token. And the token library shouldn't have to be block specific, we might want to use it for non-block things.

You could argue that lms_user_id is the same sort of thing but it felt fundamental enough to include here.

Reply:
Okay cool!